enclave/core/sgx/calls.c

```diff
@@ -131,6 +134,51 @@ extern bool oe_disable_debug_malloc_check;
 **==============================================================================
 */
 
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+extern const volatile oe_sgx_enclave_properties_t oe_enclave_properties_sgx;
+extern volatile oe_sgx_enclave_properties_t oe_enclave_properties_sgx_mutable;
+extern volatile int _have_eeid;
+
+static oe_result_t _eeid_expand()
+{
+    oe_result_t r = OE_OK;
+    uint64_t* heap_start = (uint64_t*)__oe_get_heap_base();
+
+    if (*heap_start == 0xEE1DEE1DEE1DEE1D)
```

Comment:
Please explain where the magic number number is coming from. We should define the magic number as a macro instead of spreading it around different files.

Reply:
There's nothing to explain - I just needed some magic value to recognize the fact that EEID was enabled during enclave creation (without patching the image). It's in a macro now.